src/docs/product/sentry-basics/grouping-and-fingerprints/index.mdx

```diff
@@ -36,6 +36,8 @@ Fortunately you can change the default grouping behavior to make certain issue t
 
 ## Solutions
 
+<Include name="only-error-issues-note.mdx" />
```

Comment:
This is a good spot to not use this generic message but to be a bit more descriptive. 
```suggestion
<Note>

The solutions described in this section are only applicable for [error issues](/product/issues/#error-issues). Other categories of issues (such as, [performance issues](/product/issues/#performance-issues)) do not support this feature.

</Note>
```